v2/openapi/fb-unified-openapi.yaml

```diff
@@ -4430,4 +4750,199 @@ components:
+          $ref: '#/components/schemas/PositiveAmount'
+    RampRequest:
+      allOf:
+        - $ref: '#/components/schemas/CommonRampRequestProperties'
+        - oneOf:
+            - $ref: '#/components/schemas/OnRampProperties'
+            - $ref: '#/components/schemas/OffRampProperties'
+            - $ref: '#/components/schemas/BridgeProperties'
+    CommonRamp:
+      type: object
+      required:
+        - id
+        - status
+        - createdAt
+        - updatedAt
+      properties:
+        id:
+          type: string
+        createdAt:
+          type: string
+          format: date-time
+        updatedAt:
+          type: string
+          format: date-time
+        status:
+          type: string
+          enum:
+            - Created
+            - PendingDelivery
+            - DeliveryReceived
+            - ReceiptSubmitted
+            - Completed
+            - Failed
+        amount:
+          $ref: '#/components/schemas/PositiveAmount'
+    OnRampTransfer:
+      allOf:
+        - $ref: '#/components/schemas/CommonRamp'
+        - $ref: '#/components/schemas/OnRampProperties'
+        - type: object
+          required:
+            - deliveryInstructions
+          properties:
+            deliveryInstructions:
+              $ref: '#/components/schemas/FiatDepositDestination'
+            delivery:
+              $ref: '#/components/schemas/FiatTransfer'
+            receipt:
+              $ref: '#/components/schemas/PublicBlockchainTransaction'
+    OffRampTransfer:
+      allOf:
+        - $ref: '#/components/schemas/CommonRamp'
+        - $ref: '#/components/schemas/OffRampProperties'
+        - type: object
+          required:
+            - deliveryInstructions
+          properties:
+            deliveryInstructions:
+              $ref: '#/components/schemas/PublicBlockchainAddress'
+            delivery:
+              $ref: '#/components/schemas/PublicBlockchainTransaction'
+            receipt:
+              $ref: '#/components/schemas/FiatTransfer'
+    BridgeTransfer:
+      allOf:
+        - $ref: '#/components/schemas/CommonRamp'
+        - $ref: '#/components/schemas/BridgeProperties'
+        - type: object
+          required:
+            - deliveryInstructions
+          properties:
+            deliveryInstructions:
+              $ref: '#/components/schemas/PublicBlockchainAddress'
+            delivery:
+              $ref: '#/components/schemas/PublicBlockchainTransaction'
+            receipt:
+              $ref: '#/components/schemas/PublicBlockchainTransaction'
+    Ramp:
+      oneOf:
+        - $ref: '#/components/schemas/OnRampTransfer'
+        - $ref: '#/components/schemas/OffRampTransfer'
+        - $ref: '#/components/schemas/BridgeTransfer'
   securitySchemes: {}
+  requestBodies:
+    RampRequestBody:
+      description: Ramp details
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/RampRequest'
```

Comment:
Inline it. Until now we avoided defining request bodies separately. No real value imho.